examples/audio/spatial_audio.rs

```diff
@@ -0,0 +1,94 @@
+//! This example illustrates how to load and play an audio file, and control where the sounds seems to come from.
+use bevy::prelude::*;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup)
+        .add_system(update_positions)
+        .run();
+}
+
+fn setup(
+    mut commands: Commands,
+    asset_server: Res<AssetServer>,
+    audio: Res<Audio>,
+    audio_sinks: Res<Assets<SpatialAudioSink>>,
+    mut meshes: ResMut<Assets<Mesh>>,
+    mut materials: ResMut<Assets<StandardMaterial>>,
+) {
+    let music = asset_server.load("sounds/Windless Slopes.ogg");
+    let handle = audio_sinks.get_handle(audio.play_spatial_with_settings(
+        music,
+        PlaybackSettings::LOOP,
+        Transform::IDENTITY,
+        4.0,
+        Vec3::ZERO,
+    ));
+    commands.insert_resource(AudioController(handle));
+
+    // left ear
```

Comment:
If I understand correctly, what makes it the left ear is that its translation corresponds to "half gap" from the emitter, which is set inside `audio.play_with_spatial_settings()`? (and same with the right ear).

I was confused at the beginning about "what does it make it the left ear?".

In my opinion, it would be clearer if we declare `let gap = 4.0;` at the beginning of this system (or a global const outside of it?) and then pass `gap` to `audio.play_with_spatial_settings()` and `-gap / 2.` slash `gap / 2.` to the translation of the left and right cubes.